packages/live-share/src/EphemeralEvent.ts

```diff
@@ -167,32 +168,7 @@ export interface IEphemeralEventEvents<TEvent extends IEphemeralEvent> extends I
      * @returns True if the received event is newer then the current event and should replace the current one. 
      */
     public static isNewer(current: IEphemeralEvent|undefined, received: IEphemeralEvent, debouncePeriod = 0): boolean {
```

Comment:
>IEphemeralEvent [](http://example.com/codeflow?start=35&length=15)

I think it might be cleaner to just derive IEphemeralEvent from IClientTimestamp and then make EphemeralEvent.isNewer() take two IClientTimestamp instances...